src/main/fc/settings.yaml

```diff
@@ -144,6 +144,9 @@ tables:
   - name: tristate
     enum: tristate_e
     values: ["AUTO", "ON", "OFF"]
+  - name: osd_crsf_lq_format
+    enum: osd_crsf_lq_format_e
+    values: ["TBS", "TYPE2"]
```

Comment:
I would change this to [code] values: ["TBS", "BF"] [/code]
Simply because TYPE2 is how Betaflight currently implements this.

Reply:
If we need to change this, I'd use values TBS and INAV.   :)

Reply:
:D sure!
I was simply referring to BF because that is the only firmware, at least I'm aware of, that uses the mode:lq format.
So why not name it like that?
But of course INAV instead of TYPE2 is already much better. TYPE2 is not related to anything and might give end user confusion.
(Silly questions like "is there also a TYPE1 or 3", just to avoid that.)

Reply:
Understood. The format is use by BF and emuFlight. 

I used Type2 because it's an option used for other settings. Maybe using TYPE1 and TYPE2 would be a better option? Which one looks better?

![image](https://user-images.githubusercontent.com/45466510/95783922-d43cff00-0ca0-11eb-884d-c45a61c53afd.png)

Reply:
Both are looking great!
But if you look at it from a beginners perspective, they probably have no clue what TBS and INAV format look like. (And try both anyway.)
Because of that my vote goes to the TYPE1/TYPE2 implementation.
Also more future proof is some #!%! creates a type3. :D

Reply:
Ok, updating the PR.